packages/contractkit/src/wrappers/BlockchainParameters.ts

```diff
@@ -12,6 +12,11 @@ export class BlockchainParametersWrapper extends BaseWrapper<BlockchainParameter
     this.kit,
     this.contract.methods.setIntrinsicGasForAlternativeFeeCurrency
   )
+
+  /**
+   * Getting the block gas limit.
+   */
+  getBlockGasLimit = proxyCall(this.contract.methods.blockGasLimit)
```

Comment:
Based on a quick look around, I think this should be
```suggestion
  getBlockGasLimit = proxyCall(this.contract.methods.blockGasLimit, undefined, valueToBigNumber)
```